qt/widgets/common/inc/MantidQtWidgets/Common/FitDomain.h

```diff
@@ -27,7 +27,7 @@ namespace MantidWidgets {
 class EXPORT_OPT_MANTIDQT_COMMON FitDomain {
 
 public:
-  FitDomain(std::string const &workspaceName, WorkspaceIndex workspaceIndex, double startX, double endX);
+  FitDomain(std::string const &workspaceName, WorkspaceIndex SpectraIndex, double startX, double endX);
```

Comment:
Can we leave this as `workspaceIndex`? This area of code only refers to a workspaceIndex and never spectraIndex so changing this variable name reduces naming consistency here